diff --git a/cmake/linux/helpers.cmake b/cmake/linux/helpers.cmake
index 92974fd..59603ba 100644
--- a/cmake/linux/helpers.cmake
+++ b/cmake/linux/helpers.cmake
@@ -26,10 +26,10 @@ function(set_target_properties_plugin target)
                SOVERSION ${PLUGIN_VERSION}
                PREFIX "")
 
-  # install(
-  #   TARGETS ${target}
-  #   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-  #   LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/obs-plugins)
+  install(
+    TARGETS ${target}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}/obs-plugins)
 
   if (${CMAKE_SIZEOF_VOID_P} EQUAL 4)
     set(OBSARCHNAME "32bit")
@@ -39,11 +39,6 @@ function(set_target_properties_plugin target)
     message(FATAL_ERROR "Unsupport architecture")
   endif()
 
-  install(
-    TARGETS ${target}
-    RUNTIME DESTINATION dist/${PLUGIN_FOLDER}/bin/${OBSARCHNAME}
-    LIBRARY DESTINATION dist/${PLUGIN_FOLDER}/bin/${OBSARCHNAME})
-
   if(TARGET plugin-support)
     target_link_libraries(${target} PRIVATE plugin-support)
   endif()
@@ -72,14 +67,9 @@ function(target_install_resources target)
       source_group("Resources/${relative_path}" FILES "${data_file}")
     endforeach()
 
-    # install(
-    #   DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/data/"
-    #   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/obs/obs-plugins/${target}
-    #   USE_SOURCE_PERMISSIONS)
-    
     install(
       DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/data/"
-      DESTINATION dist/${PLUGIN_FOLDER}/data
+      DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/obs/obs-plugins/${target}
       USE_SOURCE_PERMISSIONS)
   endif()
 endfunction()
@@ -88,9 +78,7 @@ endfunction()
 function(target_add_resource target resource)
   message(DEBUG "Add resource '${resource}' to target ${target} at destination '${target_destination}'...")
 
-  # install(FILES "${resource}" DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/obs/obs-plugins/${target})
-
-  install(FILES "${resource}" DESTINATION dist/${PLUGIN_FOLDER}/data)
+  install(FILES "${resource}" DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/obs/obs-plugins/${target})
 
   source_group("Resources" FILES "${resource}")
 endfunction()
